src/pages/editor-page/top-navbar/top-navbar.tsx

```diff
@@ -429,6 +437,37 @@ export const TopNavbar: React.FC<TopNavbarProps> = () => {
                                 </MenubarItem>
                             </MenubarContent>
                         </MenubarMenu>
+                        <MenubarMenu>
```

Comment:
Also here, until we fix the colors I think it might be a good idea not to let the user get into dark mode. We should hide the buttons & menu actions that get him into dark mode.